functions/localTriggers.js

```diff
@@ -6,11 +6,11 @@ const analyseJupyterSolution = require("./src/analyseJupyterSolution");
 const outgoingRequests = require("./src/outgoingRequest");
 const fetchGithubFiles = require("./src/fetchGithubFiles")
 
-const serviceAccount = require("./config/serviceAccountKey");
+const serviceAccount = require("./config/serviceAccountKey.json");
```

Comment:
@Brainenjii Do we follow the setup here, ie to create a `config/` folder inside the `functions/` folder, and then obtain the serviceAccountKey.json from `achievements-dev` firebase console, and `require()` it here?